### DISCUSSION TITLE
Add color-scheme utility/property (for light/dark scrollbars etc.)

### DISCUSSION THREAD

**Discussion by silveltman** [tailwindlabs/tailwindcss#11228] [Ideas]
I'm creating a website in darmode with on-page scrollbars. I want to use the native scrollbars, but the dark variant.

For this to work, I need to let the browser know that the website is in darkmode. I do this with the color-scheme property:

```
<!DOCTYPE html>
<html
  class="color-scheme-dark"
>
  <head>
  </head>
  <body>
    <style>
      .color-scheme-dark {
        color-scheme: dark;
      }
    </style>
  </body>
</html>
```

Would be useful to have a utility class for this, or maybe even better, automatically add the property to the 'dark' class when using `darkMode: 'class'`.

If I'm missing something here please let me know :)

**Comment by lukewarlow** [comment]
Fwiw you can also set this by using a meta tag in your html head.

**Reply by silveltman** [reply]
Yes I know, thanks! However, I switch between light/dark mode on a per-section basis, not per-page

**Comment by lukewarlow** [comment]
https://github.com/tailwindlabs/tailwindcss/pull/11271 - I've submitted a PR to add this.
